judo-runtime-core-jsl-itest/models/SpecialCasesModel/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/SpecialCasesTest.java

```diff
@@ -2149,4 +2168,209 @@ void testSafeFieldNames() {
 
     }
 
+    @Inject
+    RefDao refDao;
+
+    @Inject
+    PrimUpperDao primUpperDao;
+
+    @Inject
+    RelUpperDao relUpperDao;
+
+    @Inject
+    RelMultiUpperDao relMultiUpperDao;
+
+    @Inject
+    CompUpperDao compUpperDao;
+
+    @Inject
+    CompMultiUpperDao compMultiUpperDao;
+
+    @Test
+    @TestCase("UpperFieldNames")
+    @Requirement(reqs = {
+            "REQ-TYPE-001",
+            "REQ-TYPE-004",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-ENT-004",
+            "REQ-ENT-005",
+            "REQ-ENT-007",
+            "REQ-MDL-001",
+            "REQ-MDL-002",
+            "REQ-MDL-003",
+    })
+    void testUpperFieldNames() {
+
+        PrimUpper primUpper = primUpperDao.create(PrimUpperForCreate.builder()
+                .withAbc("Abc")
+                .withABd("ABc")
```

Comment:
This is misleading, test data should mirror the attribute IMO.

Reply:
[c88c8c2](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/294/commits/c88c8c230544f4be6a02b3157afcc721923416a5)